test/e2e/integration/LandingPage/AnchorLink.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Anchor link
```

Comment:
@tim-yao I probably no need to create a whole new feature for these kinds of tests, this can just be a scenario on the landing page test (once that has been created)